README.md

```diff
@@ -1,4 +1,30 @@
-triples
+Triples
 =======
 
+The challenge is to find 10 sets of 3 matching cards.
+
+Great workout for the memory!
+
+How to use?
+-----------
+
+Triples is part of the Sugar desktop. Please refer to;
```

Comment:
Triples is not in the Fructose set of activities, so is not part of the Sugar desktop.  The wording we use is "$N is not part of the Sugar desktop, but is often added."